homework/index.js

```diff
@@ -16,32 +20,157 @@
     xhr.send();
   }
 
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach(key => {
-      const value = options[key];
+  function createAndAppend(tag, parent, attributes = {}) {
+    const child = document.createElement(tag);
+    parent.appendChild(child);
+    Object.keys(attributes).forEach(key => {
+      const value = attributes[key];
```

Comment:
Are the names changes really necessary? If you left the code as is, these changes would have appeared as changes in the pull request, and the reviewer would not have to review these trivial changes.

Reply:
That's right. It was not necessary to change the ready functions which was already written. I didn't think that point for the reviewer, sorry. It's because maybe I am obssesive about namings 😐